### PR TITLE
Start of OSI configurations

### PR DESCRIPTION
**Pull request by Matthewkoziel**
osi.conf
osi.region.conf

### CODE REVIEW

**Comment by ClementDeltel** [comment]
It seems that this branch has some conflicts that must be resolved before merge. Please review Matthew

File: reference_guides/openframe_config/README.md [hunk 1]
```diff
@@ -2250,6 +2250,30 @@ This server is responsible for recording OSC application transaction logs. It co
 
 This server runs OpenFrame Manager. It must be set up if OpenFrame Manager will be used.
 
+## 2.1.2 OSC Sequence Configuration (ofc.ofsys.seq)
```
[the remark this answers is not in the document]
Comment (Matthewkoziel) [reply reference_guides/openframe_config/README.md]: Typo. osc.ofsys.seq is correct

**Comment by Matthewkoziel** [comment]
Resolved conflicts

**Comment by ClementDeltel** [comment]
Approved since conflict issues solved

**Pull request merged**